src/main/kotlin/no/nav/familie/ef/sak/config/ApplicationConfig.kt

```diff
@@ -53,10 +58,55 @@ class ApplicationConfig {
         return filterRegistration
     }
 
+    //Overskrever felles sin som bruker proxy, som ikke skal brukes på gcp
+    @Bean
+    @Primary
+    fun restTemplateBuilder(): RestTemplateBuilder {
+        return RestTemplateBuilder()
+                .setConnectTimeout(Duration.of(2, ChronoUnit.SECONDS))
+                .setReadTimeout(Duration.of(120, ChronoUnit.SECONDS))
+    }
+
     @Bean("utenAuth")
     fun restTemplate(restTemplateBuilder: RestTemplateBuilder,
                      consumerIdClientInterceptor: ConsumerIdClientInterceptor): RestOperations {
-        return restTemplateBuilder.additionalInterceptors(consumerIdClientInterceptor,
-                                                          MdcValuesPropagatingClientInterceptor()).build()
+        return restTemplateBuilder
+                .additionalInterceptors(consumerIdClientInterceptor,
+                                        MdcValuesPropagatingClientInterceptor()).build()
+    }
+
+    @Bean
+    fun apiKeyInjectingClientInterceptor(@Value("\${PDL_APIKEY}") pdlApiKey: String,
```

Comment:
what ? :P

Reply:
For å kommunisere med PDL må vi gå genom gateway, og då må vi sende med en apikey i en header

Det samme oppsettet finnes mellom sbs -> fss on-prem i soknad-api